modules/core/client/app/config.js

```diff
@@ -37,14 +38,21 @@ import ngreact from 'ngreact';
 
 import 'angular-waypoints/dist/angular-waypoints.all';
 
+// eslint-disable-next-line angular/window-service
+const SENTRY_DSN = window.SENTRY_DSN;
+
+if (SENTRY_DSN) {
+  require('@/config/client/sentry').init(SENTRY_DSN);
```

Comment:
Should we have this on top of the file before other imports?

Reply:
I think dynamic imports should be after static imports, logic is that the static imports can never be conditionally imported or not, so not subject to the logic of the file, and to me the logic begins where the static imports end...

... but I think the counterpoint you are maybe thinking about, is what if there is an error during the import process? ... they are all library imports though so should not be a problem.

I would just keep it like this for now... to get it merged.